AgentDiscoveries-Backend/.gitignore

```diff
@@ -1,2 +1,5 @@
 target/
 .idea
+pom.xml
+src/main/resources/META-INF/persistence.xml
+src/main/resources/config.properties
```

Comment:
These are sensible suggestions, but unfortunately some nuances mean we can't quite do it this way.

pom.xml describes which packages are needed, and so need to be included and not ignored.

persistence.xml and config.properties are files that are needed to successfully build the application, and so also need to be included and not ignored